### ISSUE TITLE
Impossible to set CheckBox.indeterminate property via API

### ISSUE THREAD

**Issue by fsmorygo** [rjaros/kvision#372]
It is still possible to access element directly, but this is not very neat
```
selectAllCheckboxControl.setValue(true)
(selectAllCheckboxControl.input.getElement() as HTMLInputElement).indeterminate = false
```

**Comment by rjaros** [comment]
I have implemented `indeterminate` property for `CheckBox` components (both readable and writable).

I've also created new `TriStateCheckBox` component, which (unlike normal `CheckBox`) supports nullable `Boolean?` value  and allows you to cycle through all three values (`true`, `false`, `null`) with the UI (mouse clicks or keyboard).

**Comment by rjaros** [comment]
Released with 5.10.0
